packages/react-ui/internal/Popup/Popup.tsx

```diff
@@ -332,8 +334,15 @@ export class Popup extends React.Component<PopupProps, PopupState> {
     }
   };
 
+  private calculateMenuWidth = (menuWidth: PopupProps['width']) => {
```

Comment:
Название метода и аргумента тоже можно поменять =)
```suggestion
  private calculateWidth = (width: PopupProps['width']) => {
```